rune/libenclave/internal/runtime/pal/pal_linux.go

```diff
@@ -5,6 +5,10 @@ import (
 	"os"
 )
 
+const (
+	currentPalApiVersion = 2
+)
+
```

Comment:
Just move the definition of pal api version from api_linux.go to pal.go and define it to 2.